samtranslator/model/eventsources/push.py

```diff
@@ -607,6 +610,61 @@ def _add_swagger_integration(self, api, function):
                 editor.add_request_model_to_method(path=self.Path, method_name=self.Method,
                                                    request_model=self.RequestModel)
 
+        if self.RequestParameters:
+
+            default_value = {
+                'Required': False,
+                'Caching': False
+            }
+
+            parameters = []
+            for parameter in self.RequestParameters:
+
+                if isinstance(parameter, dict):
+
+                    parameter_name, parameter_value = next(iter(parameter.items()))
+
+                    if not re.match('method\.request\.(querystring|path|header)', parameter_name):
```

Comment:
Should add a trailing period so strings like `method.request.pathmoreletters` isn't considered valid.

```suggestion
                    if not re.match('method\.request\.(querystring|path|header)\.', parameter_name):
```